src/service/mutator-impl.js

```diff
@@ -195,9 +200,15 @@ export class MutatorImpl {
    * @param {!Element} element
    * @param {?function()} measurer
    * @param {function()} mutator
+   * @param {boolean} skipMeasure
    * @return {!Promise}
    */
-  measureMutateElementResources_(element, measurer, mutator) {
+  measureMutateElementResources_(
+    element,
+    measurer,
+    mutator,
+    skipMeasure = false
```

Comment:
Kinda confusing since there's also a "measure" operation here.

How about `skipRemeasure`? Or `skipMeasureAfterMutate`. Or something. :)

Reply:
I like `skipRemeasure`. I'll update and then 🚢